scripts/tokenomics-emulation/config.json

```diff
@@ -83,6 +83,14 @@
         "name": "existential-deposit-transfer",
         "amount": 1
       },
+      {
+        "name": "deploy-cere-smart-contract",
+        "code_hash": "0x343f41b8cf072966ce537da45c0a9ca6a95db97dcd4e076c8a6fab99e7742023",
```

Comment:
What is it? We can not hardcode any hashes.

Reply:
These are code_hash to deploy the smart contract